Failing:

```
java.lang.AssertionError: ceaa0c895343616d4c126bdbc8b0445b
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at hudson.cli.ClientAuthenticationCacheTest.getPropertyKey(ClientAuthenticationCacheTest.java:115)
```

--- a/test/src/test/java/hudson/cli/ClientAuthenticationCacheTest.java
+++ b/test/src/test/java/hudson/cli/ClientAuthenticationCacheTest.java
@@ -112,7 +112,7 @@ public class ClientAuthenticationCacheTest {
         assertEquals(r.getURL().toString(), cache.getPropertyKey());
         JenkinsLocationConfiguration.get().setUrl(null);
         String key = cache.getPropertyKey();
-        assertTrue(key, Secret.decrypt(key) != null);
+        assertEquals(r.jenkins.getLegacyInstanceId(), key);
     }
     
     @Test

